### DISCUSSION TITLE
Input field follow up component

### DISCUSSION THREAD

**Discussion by pamelahu** [twilio-labs/paste#2081] [Q&A] [answered]
Is there currently a component/pattern for after a user selects an option in an input field and there's an immediate follow up? Example (gray box below):

![Screen Shot 2021-11-29 at 2 05 44 PM](https://user-images.githubusercontent.com/18605252/143927633-1cd1ec27-bb8b-4637-9d47-15c015683bfc.png)

**Comment by kylegilbreath** [comment]
Hi @pamelahu! Thanks for the question. Could you provide some more context on what you are trying to achieve here? Will the follow up content provide more information on what their selection means? Or were you hoping to add more options based on their answers? This feels like a great topic to bring to our office hours if you are able to attend!

**Comment by pamelahu** [comment]
> Will the follow up content provide more information on what their selection means?

So in the screenshot above, yes. I'm working on a painted door test and want to acknowledge the user's answer choice.

>  Or were you hoping to add more options based on their answers?

In a different use case (screenshot below), the gray box content depends on what users select in the dropdown field.
![Screen Shot 2021-11-30 at 5 19 57 PM](https://user-images.githubusercontent.com/18605252/144137373-a15f5a31-3a2e-4b40-ab24-43ee711e926a.png)


Basically, both of these gray boxes are dependent on the selection made by the user in the input field right above it. The gray box is just what I've created so far. With the gray box, I wanted it to be "indented" to illustrate its hierarchy is under the initial input field and that it is a follow-up to what the user just selected. I'm very flexible on the UI and I also didn't want to reinvent the wheel - so that's why I'm asking if there's an existing way to handle this.

> This feels like a great topic to bring to our office hours if you are able to attend!

Unfortunately there aren't any slots this week and I need to ship this soon. :(

**Comment by loreina** [comment, marked as answer]
Hi @pamelahu - just wanted to share the office hours recording with you in case you'd like to revisit any of our conversations. [Here's the video](https://www.loom.com/share/e37ae97022b543b78ba79a3e70e09f14) (it includes timestamps for each topic), and the password is posted in the #help-design-system channel. 

To summarize some of our conversation:
- We discussed separating "Select Organization Type" and "Submit Application" sections
- We explored some design ideas for the nested input fields based on the organization type

Thanks again for coming to office hours!
